src/components/elements/generatedStoryText.tsx

```diff
@@ -1,9 +1,41 @@
 export const GeneratedStoryText = (props: any) => {
-    const story = props.story;
-    return (
-      <>
-      
-        Generated Story Text
-      </>
-    );
+  const story = props.story;
+  const total_manuscripts_num = 481;
+ 
+  // function to determine story age
+  const DetermineStoryAge = (earliest_attest: number) => {
+    let age;
+   
+    if (earliest_attest == 14 || earliest_attest == 15) {
```

Comment:
I think you should change the comparison operator to === from ==. == only compares value, but === compares both value and type so it's safer. Other than that I like your approach, it makes sense to make functions for the different logic.